cmd/server/main.go

```diff
@@ -1,3 +1,48 @@
 package main
 
-func main() {}
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+
+	h "github.com/ArtemShalinFe/metcoll/cmd/server/internal/handlers"
+	"github.com/caarlos0/env"
+)
+
+type Config struct {
+	Address string `env:"ADDRESS" envDefault:"localhost:8080"`
+}
+
+func main() {
+
+	cfg := parseConfig()
+
+	r := h.ChiRouter()
+
+	fmt.Printf("Try running on %v\n", cfg.Address)
+	err := http.ListenAndServe(cfg.Address, r)
```

Comment:
Вопрос стиля, но почему бы не:

```suggestion
	if err := http.ListenAndServe(cfg.Address, r); err != nil {
```

Reply:
Это не единственное место, где `err` можно было бы обернуть в `if`. Это не только, кстати, вопрос стиля, но и вопрос области видимости переменной с ошибкой, которую ты создаешь. Ну и в целом лучше придерживаться единного стиля

Reply:
Заменил, где нашел.

Reply:
Прошелся по коду, на мой взгляд мест больше нет, где можно было бы поправить.